src/options.ts

```diff
@@ -46,7 +46,7 @@ export type Options = {
   fileSuffix: string;
   importSuffix: string;
   outputEncodeMethods: true | false | "encode-only" | "decode-only" | "encode-no-creation";
-  outputJsonMethods: boolean;
+  outputJsonMethods: true | false | "encode-only" | "decode-only";
```

Comment:
@Frabat wdyt about calling these `"to-only"` and `"from-only"`, given that the methods that are output are `toJSON` / `fromJSON`.

Reply:
I'm completely ok with that! I wanted to keep the same options as the outputEncodeMethods field, but that's totally doable :D 

Will update the readme too.